lte/gateway/c/core/oai/test/spgw_task/data/data_list.txt

```diff
@@ -0,0 +1,100 @@
+/home/vagrant/magma/lte/gateway/c/core/oai/test/spgw_task/data/IMSI001010000000066_SPGW
```

Comment:
since `/home/vagrant/magma/lte/gateway/c/core/oai/test/spgw_task/data/` is constant, how about just define only the name of the files here? and then pass and append the folder path on the cpp file instead?

Reply:
This is a good idea! My intention at that time is to allow more dynamic way to specify the path of samples. But if it is not necessary, we can do as your suggestions.